.github/workflows/run-acceptance-tests.yml

```diff
@@ -1,7 +1,6 @@
 # WARNING: This file is autogenerated - changes will be overwritten if not made via https://github.com/pulumi/ci-mgmt
 
 env:
-  VERSION_PREFIX: 4.0.0
```

Comment:
We are removing the version override, which means codegen will use the tag version. This will fail until we release `v4.0.0`.